es6-const-let/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>es6-const-let</title>
+</head>
+<body>
+  <script>
+    const greetings = [
+      'hello!',
+      'konichiwa!',
+      'nihao',
+      'bonjour',
+      'gutten tag',
+      'ola'
+    ];
+
+    function logAllGreetings(greetingList) {
+      let i = 0;
+      while (i < greetingList.length) {
+        console.log('Current greeting:', greetingList[i]);
+        i++;
+      }
+    }
+    logAllGreetings(greetings);
+
+    function getRandomGreeting(greetingList) {
+      let randomIndex = Math.floor(Math.random() * greetingList.length);
+      let greeting = greetingList[randomIndex];
+      if (randomIndex === 1) {
+        greeting = greeting + ' is a japanese greeting.';
+      }
+      return greeting;
+    }
+    console.log('Random greeting:', getRandomGreeting(greetings));
+
+    function addToGreetings(newGreeting, greetingList) {
+      greetingList.push(newGreeting);
+    }
+    addToGreetings('namaste', greetings);
+    logAllGreetings(greetings);
+
+    let variable = 'meow';
```

Comment:
This one should also be a `const` The reason for this one and the above one being consts is that their values are never reassigned.

Reply:
Fixed above two variables accordingly. Thanks.
![image](https://user-images.githubusercontent.com/76827487/118039956-c67a5580-b325-11eb-96ae-cfff317fc3a1.png)